src/mumble/Settings.cpp

```diff
@@ -654,13 +649,15 @@ BOOST_TYPEOF_REGISTER_TYPE(QVariant)
 BOOST_TYPEOF_REGISTER_TYPE(QFont)
 BOOST_TYPEOF_REGISTER_TYPE(EchoCancelOptionID)
 BOOST_TYPEOF_REGISTER_TEMPLATE(QList, 1)
+*/
 
+#define TYPEOF(var) std::remove_reference<decltype(var)>::type
```

Comment:
To be on the safe side, this should also `remove_const`. Also note that you can make use of the `*_t` shorthands as we have moved to C++17 by now